lambdas/process-journey-event/src/main/java/uk/gov/di/ipv/core/processjourneyevent/statemachine/TransitionResult.java

```diff
@@ -12,6 +14,6 @@ public record TransitionResult(
         Map<String, String> auditContext,
         String targetEntryEvent) {
     public TransitionResult(State state) {
-        this(state, null, null, null);
+        this(state, new ArrayList<>(), new HashMap<>(), null);
```

Comment:
`List.of()` (or `Collections.emptyList()`) and the equivalent map, are fractionally more performant (saves an allocation). Doesn't really matter for our performance, but perhaps a good habit!

(Fine to leave if you prefer)

Reply:
Nice, thanks for the tip